hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ReplicationManager.java

```diff
@@ -1028,6 +1047,8 @@ private void sendReplicateCommand(final ContainerInfo container,
     LOG.info("Sending replicate container command for container {}" +
             " to datanode {} from datanodes {}",
         container.containerID(), datanode, sources);
+    metrics.incrNumReplicateCmdsSent();
```

Comment:
Suggest to move the metrics operation after the command is sent.

Reply:
Yes, it should be updated after sent.